addOns/ascanrulesAlpha/src/main/resources/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties

```diff
@@ -30,3 +30,7 @@ ascanalpha.mongodb.extrainfo.alldata=In some PHP or NodeJS based back end implem
 ascanalpha.mongodb.extrainfo.sleep=Through the where or group MongoDB clauses, Javascript sleep function is probably executable.
 ascanalpha.mongodb.extrainfo.json=In some NodeJS based back end implementations, messages having the JSON format as content-type are expected. In order to obtain sensitive data it is possible to attack these applications injecting the "{$ne:}" string (or other similar ones) that is processed as an associative array rather than a simple text.\nThrough this, the queries made to MongoDB will always be true.
 ascanalpha.mongodb.extrainfo.authbypass=More probably a MongoDB attack has bypassed an authentication page.
+
+ascanalpha.forbiddenBypass.name=Bypassing 403
+ascanalpha.forbiddenBypass.desc=Bypassing 403 endpoints may be possible.
```

Comment:
To be more explicit it could be added e.g.: `, the scan rule sent a payload that caused the response to be accessible (status code 200).`

Reply:
Done